src/CP/core/utils/stopwatch.jl

```diff
@@ -7,6 +7,7 @@ Start a timer.
 function tic()
     t0 = time_ns()
     task_local_storage(:TIMERS, (t0, get(task_local_storage(), :TIMERS, ())))
+    nothing
```

Comment:
This function was type unstable because Julia was not able to infer the returned type. As this function is not supposed to return something, I just added it explicitly to make life a little bit easier to the compiler even though I I guess the performance impact is almost null.